tst/EnergyPlus/unit/AirTerminalSingleDuct.unit.cc

```diff
@@ -58,22 +58,43 @@
 
 #include "Fixtures/EnergyPlusFixture.hh"
 
+#include <EnergyPlus/DataAirLoop.hh>
+#include <EnergyPlus/DataAirSystems.hh>
+#include <EnergyPlus/DataDefineEquip.hh>
+#include <EnergyPlus/DataEnvironment.hh>
 #include <EnergyPlus/DataGlobals.hh>
+#include <EnergyPlus/DataHVACGlobals.hh>
+#include <EnergyPlus/DataHeatBalFanSys.hh>
+#include <EnergyPlus/DataLoopNode.hh>
+#include <EnergyPlus/DataZoneEnergyDemands.hh>
 #include <EnergyPlus/DataZoneEquipment.hh>
 #include <EnergyPlus/HVACSingleDuctInduc.hh>
 #include <EnergyPlus/HeatBalanceManager.hh>
+#include <EnergyPlus/MixedAir.hh>
+#include <EnergyPlus/OutAirNodeManager.hh>
+#include <EnergyPlus/Psychrometrics.hh>
 #include <EnergyPlus/ScheduleManager.hh>
 #include <EnergyPlus/SingleDuct.hh>
 #include <EnergyPlus/ZoneAirLoopEquipmentManager.hh>
 
 // EnergyPlus Headers
+using namespace EnergyPlus::DataAirLoop;
+using namespace EnergyPlus::DataAirSystems;
+using namespace EnergyPlus::DataEnvironment;
 using namespace EnergyPlus::DataGlobals;
+using namespace EnergyPlus::DataHVACGlobals;
 using namespace EnergyPlus::DataZoneEquipment;
 using namespace EnergyPlus::HeatBalanceManager;
 using namespace EnergyPlus::HVACSingleDuctInduc;
+using namespace EnergyPlus::DataLoopNode;
+using namespace EnergyPlus::MixedAir;
+using namespace EnergyPlus::Psychrometrics;
 using namespace EnergyPlus::ScheduleManager;
 using namespace EnergyPlus::SingleDuct;
 using namespace EnergyPlus::ZoneAirLoopEquipmentManager;
+using namespace EnergyPlus::DataDefineEquip;
+using namespace EnergyPlus::DataZoneEnergyDemands;
+using namespace EnergyPlus::DataHeatBalFanSys;
```

Comment:
Just an FYI, we are moving away from any "using" statements and instead including the namespace with the class/struct variable (e.g., DataLoopNode::Node(1).Temp). I'm leaving this alone for now because it takes some time to make all the changes.